### PR TITLE
Fix graph refresh example in dummy provider

### PR DESCRIPTION
**Pull request by agrare**
This updates the dummy provider to the new graph refresh interface

### CODE REVIEW

File: app/models/manageiq/providers/dummy_provider/inventory/persister/cloud_manager.rb [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+  end
 
-  has_cloud_manager_vms
+  def parent
+    manager.presence
+  end
+
+  def shared_options
+    {
+      :parent => parent
```
Comment (agrare) [inline app/models/manageiq/providers/dummy_provider/inventory/persister/cloud_manager.rb]: @slemrmartin it seems like without this https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/save_collection/saver/base.rb#L12 will blow up here https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/inventory_collection.rb#L964
Is there any reason this isn't in core?

**Comment by agrare** [comment]
cc @Ladas @Fryguy

File: app/models/manageiq/providers/dummy_provider/inventory/persister/cloud_manager.rb (hunk 1 above)
Comment (slemrmartin) [reply app/models/manageiq/providers/dummy_provider/inventory/persister/cloud_manager.rb]: @agrare previously I added missing `parent ` in `add_collection`.
We can add it to core to `shared_options`, no problem.

Comment (agrare) [reply app/models/manageiq/providers/dummy_provider/inventory/persister/cloud_manager.rb]: :+1: awesome thanks @slemrmartin

**Pull request merged**